shared/types/submission.ts

```diff
@@ -87,3 +94,31 @@ export type FormSubmissionMetadataQueryDto = RequireAtLeastOne<
   },
   'page' | 'submissionId'
 >
+
+type SubmissionContentBase = {
+  responses: FieldResponse[]
+}
+
+/**
+ * Shape of email form submissions
+ */
+export type EmailModeSubmissionContentDto = SubmissionContentBase
```

Comment:
tiny nit but this is really a mouthful - any issue with the old `EmailSubmissionDto` naming?

Reply:
EmailSubmissionDto gives the wrong impression that that is the entire shape of the DTO, when in actuality the content has to be transformed into a `FormData` object prior to calling the endpoint. Just thought to make that distinction. Your thoughts?

Reply:
Will fix in another PR if you think we should still change :P